### ISSUE TITLE
Add Mirror Software Graphs / Data Analysis

### ISSUE THREAD

**Issue by cooperlees** [crflynn/pypistats.org#12]
Hi,

Many thanks for this project! It's cool.

I'm a [bandersnatch](https://pypi.org/project/bandersnatch/) contributor and I'd love to see a page added for Mirroring software and the versions of each respective system used. Namely, `bandersnatch`, as I am bias.

Do you have an idea of how you'd like this done if I (or you) were to implement it? Today I query Google and a nice graph on your page would be amazing and simplify my life! :D

**Comment by crflynn** [comment]
Do you mean aggregating the downloads across all pypi specifically for mirroring applications? That could be interesting sure. Do you have some sample queries to share so I can get an idea of what specifically you would want to see?

**Comment by cooperlees** [comment]
Exactly. Getting stats like my example for each mirroring software is nice to show what version people are using around the world. Here is an example query for only bandersnatch:
```
SELECT
  details.installer.version,
  count(*)
FROM
  `the-psf.pypi.downloads20181125`
WHERE
  details.installer.name = 'bandersnatch'
GROUP BY
  1
ORDER BY
  2
```

This will show the trends of how often people update bandersnatch and other mirroring software. Should also let us roughly guess how many bandersnatch installations are running around the world each day.
